css/easyddb_status.css

```diff
@@ -0,0 +1,132 @@
+/* line 2, ../../../themes/ddbasic/sass/contrib/neat/grid/_grid.scss */
+* {
```

Comment:
This resetting really fucks up so many things. We have had bounced in several modules because of this. Why is this needed @NicolaeGrigoriu ?